### PR TITLE
Merge dam

### PR DESCRIPTION
**Pull request by LitschiW**
This change is [Reviewable](https://reviewable.io/reviews/dqualizer/dqlang/29)

### CODE REVIEW

File: src/main/java/io/github/dqualizer/dqlang/types/rqa/definition/RuntimeQualityAnalysis.java [hunk 1]
```diff
@@ -2,16 +2,19 @@
...
+import lombok.Builder;
 import lombok.Data;
+import lombok.Getter;
 
 import java.util.HashSet;
 import java.util.Set;
 
-@Data
+@Getter
+@Builder
```
Comment (haninator) [inline src/main/java/io/github/dqualizer/dqlang/types/rqa/definition/RuntimeQualityAnalysis.java:15]:
```suggestion
@Builder
@ToString
```

File: build.gradle.kts [hunk 2]
```diff
@@ -75,25 +72,33 @@ repositories {
...
-    implementation("javax.validation:validation-api:2.0.1.Final")
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.7.2")
-    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json-jvm:1.5.1")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json-jvm")
     implementation("com.google.code.findbugs:jsr305:3.0.2")
+    implementation("javax.validation:validation-api:2.0.1.Final")
 
     annotationProcessor("org.projectlombok:lombok")
```
Comment (haninator) [inline build.gradle.kts:91]:
```suggestion
    annotationProcessor("org.projectlombok:lombok:1.18.30")
```

Comment (EddeCCC) [reply build.gradle.kts]: Done

File: src/main/java/io/github/dqualizer/dqlang/types/rqa/definition/RuntimeQualityAnalysis.java (hunk 1 above)
Comment (EddeCCC) [reply src/main/java/io/github/dqualizer/dqlang/types/rqa/definition/RuntimeQualityAnalysis.java:12]: I think the previous used annotation @Data makes more sense.

**Comment by EddeCCC** [comment]
In my opinion we should move all classes to only one language. Probably Kotlin.

**Review by EddeCCC — COMMENTED** [review]
Will be done in another branch

> *[Reviewable](https://reviewable.io/reviews/dqualizer/dqlang/29#-:-NokP4bV9DM1bfZA2Gso:b-pr8fgw)* status: 0 of 153 files reviewed, 2 unresolved discussions (waiting on @DaTa and @haninator)

**Review by EddeCCC — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/dqualizer/dqlang/29#-:-NokPBtf9ISFwTXtNaop:bnfp4nl)* status: 0 of 153 files reviewed, 2 unresolved discussions (waiting on @DaTa and @haninator)

**Pull request merged**